src/dbauth/index.ts

```diff
@@ -95,11 +95,13 @@ export class DBAuth {
     designDocs?: any[],
     type?: string,
     adminRoles?: string[],
-    memberRoles?: string[]
+    memberRoles?: string[],
+    partitioned?: boolean
```

Comment:
While fixing this, it came to me that the code does not reflect what is described in this paragraph : the user facing method `addUserDB` in user.ts has actualy :

``public addUserDB(
    login: string,
    dbName: string,
    type: 'private' | 'shared' = 'private',
    designDocs?
  ) {``

  while the doc lets suggest that one may pass a permission array.

Reply:
Yeah sure, feel free to fix that up. I really need to migrate to TypeDoc instead of having to update this readme...

And sorry, I didn't mean „remove the lockfile” but „remove the changes, keeping the lockfile unchanged“ ;)

Also: do you have to add the `partitioned` parameter in the test of `DBAuth`? I think it should do the default (not partitioned) correctly when not adding the parameter.

Reply:
Oops its me, the lock file is back there, as it was before the PR (github does not display it quite correctly though)

And yes, the partitioned parameter is not mandatory, so it is false by default